spec/features/bulk_updates/bulk_updates_search_spec.rb

```diff
@@ -22,7 +22,7 @@
 
     it 'displays the correct options for the search field' do
       within '#collection-search' do
-        options = BulkUpdatesHelper::SEARCHABLE_KEYS.map { |key, value| value[:title] }
+        options = BulkUpdatesHelper::SEARCHABLE_KEYS.map { |_key, value| value[:title] }
 
         expect(page).to have_select('field', options: options)
       end
```

Comment:
comment for line 31 (but the closest github is letting me get is 28):
Should have thought of this earlier (sorry) - should we have a test like this for the `require_from_group` validation rule you added for the `double_date` fields?

Reply:
Probably a good idea